### DISCUSSION TITLE
title: and description: yaml properties in markdown files

### DISCUSSION THREAD

**Discussion by pitosalas** [GitbookIO/community#204] [0. General bugs and questions]
I discovered that if I attach yaml to a README.md file (and any other file) the title and description are displayed beautifully as part of gitbooks standard template.

1. What other yaml properties might be implemented in the template?
2. How can I supply my own page template?

Thanks,
## Here's what the .md file looks like
<img width="844" alt="Screenshot 2023-05-13 at 6 38 08 PM" src="https://github.com/GitbookIO/community/assets/44681/68e8c38c-e241-4885-8f57-4eeff319fe1a">

## Here's what the resulting web page looks like
<img width="641" alt="Screenshot 2023-05-13 at 6 37 39 PM" src="https://github.com/GitbookIO/community/assets/44681/a1a13e91-7ec0-407d-ae2a-8f51bb85f036">

**Comment by lharris-te** [comment]
Dear GitBook, I'd love to see additional examples of this! 
I'd be interested to see
1. What does the source file look like
2. What kind of syntax is allowed
3. How is the file rendered when live
